### PR TITLE
Add hello reshape ssd sample

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: samples/js/node/notebooks/213-question-answering.nnb [hunk 1]
```diff
@@ -58,25 +58,21 @@
...
+                "const core = new ov.Core();\nconst model = core.readModel(modelXMLPath);\n\nnew ov.PrePostProcessor(model)\n  .setInputElementType(0, ov.element.f32)\n  .setInputElementType(1, ov.element.f32)\n  .setInputElementType(2, ov.element.f32)\n  .setInputElementType(3, ov.element.f32)\n  .build();\n\nconst compiledModel = core.compileModel(model, 'CPU');\n\nconst inputs = compiledModel.inputs;\nconst outputs = compiledModel.outputs;\n\nconst inputSize = compiledModel.input(0).shape[1];"
             ],
             "outputs": []
         },
         {
             "language": "javascript",
             "source": [
-                "inputs.forEach(i => console.log(`${i}`));\nconsole.log('===')\noutputs.forEach(o => console.log(`${o}`));"
```
Comment (almilosz) [inline samples/js/node/notebooks/213-question-answering.nnb:68]: I think we can list both inputs and outputs:
`inputs.forEach(i => console.log(`${i}`));`

File: samples/js/node/hello_classification/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Hello Classification NodeJS Sample
 
 Run:
-`node hello_classification.js ../../assets/models/v3-small_224_1.0_float.xml ../../assets/images/coco224x224.jpg AUTO`
+`node hello_classification.js ../../assets/models/classification.xml ../../assets/images/coco.jpg AUTO`
```
Comment (almilosz) [inline samples/js/node/hello_classification/README.md:4]: Where can I find this classification.xml model?

File: samples/js/node/hello_reshape_ssd/hello_reshape_ssd.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+  //----------------- Step 5. Loading model to the device ----------------------
+  console.log('Loading the model to the plugin');
+  const compiledModel = core.compileModel(model, deviceName);
+
+  //---------------- Step 6. Create infer request and do inference synchronously
+  console.log('Starting inference in synchronous mode');
+  const inferRequest = compiledModel
+    .createInferRequest();
+  // FIXME: doesn't work
```
Comment (almilosz) [inline samples/js/node/hello_reshape_ssd/hello_reshape_ssd.js:76]: What is the error message?

**Review by almilosz — APPROVED** [review]

File: samples/js/node/notebooks/213-question-answering.nnb (hunk 1 above)
Comment (vishniakov-nikolai) [reply samples/js/node/notebooks/213-question-answering.nnb:68]: Added

File: samples/js/node/hello_classification/README.md (hunk 2 above)
Comment (vishniakov-nikolai) [reply samples/js/node/hello_classification/README.md]: README.md extended

File: samples/js/node/hello_reshape_ssd/hello_reshape_ssd.js (hunk 3 above)
Comment (vishniakov-nikolai) [reply samples/js/node/hello_reshape_ssd/hello_reshape_ssd.js]: Fixme line was removed

**Pull request merged**